src/Dialogs/ForegroundProcessDialog.vala

```diff
@@ -44,6 +44,18 @@ public class Terminal.ForegroundProcessDialog : Granite.MessageDialog {
         );
     }
 
+    public ForegroundProcessDialog.before_tab_reload (MainWindow parent) {
+        Object (
+            transient_for: parent,
+            primary_text: _("Are you sure you want to reload this tab?"),
+            secondary_text:
+                _("There is an active process on this tab.") + " " +
+                _("If you reload it, the process will end."),
```

Comment:
I don't think it good practice to construct messages like this - it can prevent proper translation into some languages (and there is no context for "it" in the second translation).
```suggestion
                _("There is an active process on this tab. If you reload it, the process will end.")
```

Reply:
It just copy-pasted this part from above, it should probably also be changed there then!